hugr-core/tests/snapshots/model__roundtrip_cfg.snap

```diff
@@ -22,10 +22,9 @@ expression: "roundtrip(include_str!(\"../../hugr-model/tests/fixtures/model-cfg.
     (cfg [%0] [%1]
       (signature (core.fn [?0] [?0]))
       (cfg [%2] [%3]
-        (signature (core.fn [(core.ctrl [?0])] [(core.ctrl [?0])]))
+        (signature (core.ctrl [[?0]] [[?0]]))
```

Comment:
Why do these need two sets of square brackets?

Reply:
The outer list is the list of branches. The inner lists are the lists of values for those branches.